src/app/zap-templates/zcl/data-model/chip/group-key-mgmt-cluster.xml

```diff
@@ -40,14 +57,9 @@ limitations under the License.
     <item fieldId="5" name="EpochStartTime1" type="epoch_us" isNullable="true"/>
     <item fieldId="6" name="EpochKey2" type="OCTET_STRING" length="16" isNullable="true"/>
     <item fieldId="7" name="EpochStartTime2" type="epoch_us" isNullable="true"/>
+    <item fieldId="8" name="GroupKeyMulticastPolicy" type="GroupKeyMulticastPolicyEnum" isNullable="false"/>
```

Comment:
We must make this optional and treat it as value 0. Otherwise we break all existing clients.